compiler/damlc/daml-stdlib-src/DA/Internal/Interface.daml

```diff
@@ -123,17 +124,10 @@ _exerciseDefault = exerciseGuarded (const True)
 --  2. when t is an interface type and i is an interface it requires
 --  3. when t and i are the same interface type
 --
--- It supports these three cases uniformly because `fromInterface`
+-- It supports these three cases uniformly because `unsafeFromInterface`
 -- handles each of them separately.
 _exerciseInterfaceGuard : forall i t.
   (HasFromInterface t i, HasInterfaceTypeRep i, HasTemplateTypeRep t) =>
   ContractId t -> (t -> Bool) -> i -> Bool
 _exerciseInterfaceGuard cid tpred ivalue =
-  case fromInterface @t ivalue of
-    Some tvalue -> tpred tvalue
-  #ifdef DAML_EXPERIMENTAL
```

Comment:
I think we can drop the CPP extension in this file now :-)

Reply:
ah yes, thanks